packages/discovery-react-components/package.json

```diff
@@ -26,13 +26,13 @@
     "dist"
   ],
   "peerDependencies": {
-    "@carbon/icons": "^10.5.0",
+    "@carbon/icons": "^10.6.1",
```

Comment:
for peer dependencies in general, we should probably specify minimum versions for our components to work https://nodejs.org/es/blog/npm/peer-dependencies/

things like `10.x.x` or `>= 10.5.0 < 11`